pkg/broker/broker.go

```diff
@@ -94,7 +179,17 @@ func (b *Broker) Provision(request *osb.ProvisionRequest, _ *broker.RequestConte
 
 	klog.V(4).Infof("broker: provisioning request %+v in namespace %q", request, namespace)
 
-	operationName, err := b.Client.Provision(request.InstanceID, request.ServiceID, request.PlanID, namespace, request.AcceptsIncomplete, request.Parameters)
+	// Check if override parameters are defined for the service to be provisioned
+	// If defined those parameters will be used instead of what the user provided
```

Comment:
```suggestion
	// Check if override parameters are defined for the service to be provisioned.
	// If defined, those parameters will be used instead of what the user provided.
```